src/utils/components/SSHAccess/utils.ts

```diff
@@ -68,9 +71,21 @@ export const createSSHService = async (vm: V1VirtualMachine, vmi: V1VirtualMachi
 
   const serviceResource = buildSSHServiceFromVM(vm, labelSelector);
 
-  await k8sCreate({
+  return await k8sCreate({
     model: ServiceModel,
     data: serviceResource,
     ns: namespace,
   });
 };
+
+export const getConsoleVirtctlCommand = (vmName: string, vmNamespace?: string) => {
+  let command = 'virtctl ';
+
+  if (vmNamespace) {
+    command += `-n ${vmNamespace} `;
+  }
+
+  command += `ssh ${vmName}`;
+
+  return command;
```

Comment:
`virtctl {$vmNamespace ? -n ${vmNamespace} : ""} ssh ${vmName}` - not sure if this function is needed

Reply:
Surely Hilda will use it in another place for the dropdown action so I thought a function would be useful